test/system/200-pod.bats

```diff
@@ -335,8 +335,15 @@ EOF
     is "$output" ".*Invalid kernel namespace to share: bogus. Options are: cgroup, ipc, net, pid, uts or none" \
        "pod test for bogus --share option"
     run_podman pod create --share ipc --name $pod_name
+    run_podman pod inspect -l --format "{{.SharedNamespaces}}"
```

Comment:
`-l` doesn't work on remote (hence the test failures). Can you change this and line 343 to use `$pod_name` instead? We already have other tests for `-l`